Homework-4-ManyToManyRelationships/src/api/groups.ts

```diff
@@ -0,0 +1,107 @@
+import express, { NextFunction, Request, Response } from "express";
+import { Container } from "typedi";
+import { GroupService } from "../services/groupService";
+import { BaseGroupDTO, GroupDTO } from "../types/groupDTO";
+
+export const groupsRouter = express.Router();
+const groupService = Container.get(GroupService);
+
+const getAllGroups = async (_req: Request, res: Response, next: NextFunction) => {
+  try {
+    const groups = await groupService.getAllGroups();
+    res.status(200).send(groups);
+  } catch (error) {
```

Comment:
I'm wondering if this error handling is anyhow different from default one?

Reply:
Will make it in homework dedicated to error handling